lib/components/tiled_component.dart

```diff
@@ -69,15 +69,19 @@ class TiledComponent extends Component {
       final image = images[tile.image.source];
 
       final rect = tile.computeDrawRect();
-      final src = Rect.fromLTWH(rect.left.toDouble(), rect.top.toDouble(),
-          rect.width.toDouble(), rect.height.toDouble());
-      final dst = Rect.fromLTWH(tile.x.toDouble(), tile.y.toDouble(),
-          rect.width.toDouble(), rect.height.toDouble());
+      final src = Rect.fromLTWH(rect.left.toDouble(), rect.top.toDouble(), rect.width.toDouble(), rect.height.toDouble());
+      final dst = Rect.fromLTWH(tile.x.toDouble(), tile.y.toDouble(), rect.width.toDouble(), rect.height.toDouble());
 
       c.drawImageRect(image, src, dst, paint);
     });
   }
 
   @override
   void update(double t) {}
+  ObjectGroup getObjectLayerByName(String name) {
```

Comment:
Insert a line above here please

Reply:
Also this method name is a little confusing for me, what do you think of `getObjectGroupFromLayer`?

Reply:
I was trying to dettach from the lib, actually I think the ObjectGroup isnt a good name, despite  objectgroup is the name of the tag in the xml, in the UI they use object layer, so I tried to look more like the UI than the XML, but I can change, no problem.